### PR TITLE
test: add test util to prefix t.Logf with date string

### PR DESCRIPTION
**Pull request by bap2pecs**
## Summary

I found this helpful at debugging. after the change, we will see sth like

```
log.go:12: [14:15:52.270] Register consumer op-stack-l2-901 to Babylon
```

## Test Plan

```
make lint
make test-e2e-op
```

### CODE REVIEW

File: itest/opstackl2/op_e2e_test.go [hunk 1]
```diff
@@ -48,7 +49,7 @@ func TestOpSubmitFinalitySignature(t *testing.T) {
 	committedPubRand, err := queryFirstPublicRandCommit(ctm.OpL2ConsumerCtrl, fpInstance.GetBtcPk())
 	require.NoError(t, err)
 	committedStartHeight := committedPubRand.StartHeight
-	t.Logf("First committed pubrandList startHeight %d", committedStartHeight)
+	log.Logf(t, "First committed pubrandList startHeight %d", committedStartHeight)
```
Comment (SebastianElvis) [inline itest/opstackl2/op_e2e_test.go:52]: Can this be done by having a wrapper function for `t.Logf` rather than introducing a new logger?

Comment (SebastianElvis) [reply itest/opstackl2/op_e2e_test.go:52]: Is there a more native way for doing it, e.g., modifying log format under `t` object?

Comment (bap2pecs) [reply itest/opstackl2/op_e2e_test.go:52]: I did some research but couldn't find it

Comment (SebastianElvis) [reply itest/opstackl2/op_e2e_test.go:52]: Did some research and it seems so :sweat:

**Review by SebastianElvis — APPROVED** [review]

**Pull request merged**